core/sail/elasticsearch-store/src/main/java/org/eclipse/rdf4j/sail/elasticsearchstore/config/ElasticsearchStoreConfig.java

```diff
@@ -0,0 +1,168 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+package org.eclipse.rdf4j.sail.elasticsearchstore.config;
+
+import org.eclipse.rdf4j.model.Model;
+import org.eclipse.rdf4j.model.Resource;
+import org.eclipse.rdf4j.model.ValueFactory;
+import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
+import org.eclipse.rdf4j.model.util.ModelException;
+import org.eclipse.rdf4j.model.util.Models;
+import org.eclipse.rdf4j.sail.base.config.BaseSailConfig;
+import org.eclipse.rdf4j.sail.config.SailConfigException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.eclipse.rdf4j.sail.elasticsearchstore.config.ElasticsearchStoreSchema.NAMESPACE;
+
+/**
+ * @author Håvard Mikkelsen Ottestad
+ */
+public class ElasticsearchStoreConfig extends BaseSailConfig {
+
+	private static final ValueFactory vf = SimpleValueFactory.getInstance();
+
+	private String hostname;
+	private int port = -1;
+	private String clusterName;
+	private String index;
+
+	public ElasticsearchStoreConfig() {
+		super(ElasticsearchStoreFactory.SAIL_TYPE);
+	}
+
+	@Override
+	public Resource export(Model graph) {
+		Resource implNode = super.export(graph);
+
+		graph.setNamespace("ms", NAMESPACE);
```

Comment:
Why `ms`? Perhaps change to something like `ess` to make conflicts with other namespace prefixes less likely.

Reply:
I think this wast copy-paste from somewhere else. Will change :)

Reply:
Copy paste from MemoryStore => "ms"